yellowbrick/classifier/classification_report.py

```diff
@@ -62,7 +62,7 @@ class ClassificationReport(ClassificationScoreVisualizer):
 
     cmap : string, default: ``'YlOrRd'``
         Specify a colormap to define the heatmap of the predicted class
-        against the actual class in the confusion matrix.
+        against the actual class in the classification report.
```

Comment:
In the docstring of the `ClassificationReport` visualizer, the `cmap` parameter had said "confusion matrix" instead of "classification report".